src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetTemplate/index.spec.tsx

```diff
@@ -12,15 +13,18 @@ const genericInfoText = 'An example info text';
 
 describe('<DashboardVideoLiveWidgetTemplate />', () => {
   it('renders DashboardVideoLiveWidgetTemplate opened, with an info text ', () => {
+    const queryClient = new QueryClient();
     render(
       wrapInIntlProvider(
-        <DashboardVideoLiveWidgetTemplate
-          initialOpenValue={true}
-          infoText={genericInfoText}
-          title={genericTitle}
-        >
-          {genericComponent1}
-        </DashboardVideoLiveWidgetTemplate>,
+        <QueryClientProvider client={queryClient}>
```

Comment:
Why do you need the `QueryClientProvider` now ? You are not using react-query in the `DashboardVideoLiveWidgetTemplate` component

Reply:
You're right, I deleted it